TILApp/Views/AuthView/SignInViewController.swift

```diff
@@ -1,27 +1,77 @@
 import UIKit
+import AuthenticationServices
 
 final class SignInViewController: UIViewController {
-    private lazy var button = {
-        let button = UIButton()
-        button.setTitle("SIGN IN", for: .normal)
-        button.addTarget(self, action: #selector(signInButtonTapped), for: .touchUpInside)
-        return button
-    }()
 
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .systemBackground
 
-        view.addSubview(button)
-        button.translatesAutoresizingMaskIntoConstraints = false
+        setAppleLoginButton()
+    }
+
+    func setAppleLoginButton() {
+        let authorizationButton = ASAuthorizationAppleIDButton(type: .signIn, style: .whiteOutline)
+        authorizationButton
+            .addTarget(self, action: #selector(handleAuthorizationAppleIDButtonPress), for: .touchUpInside)
+
+        view.addSubview(authorizationButton)
+        authorizationButton.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            button.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            button.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            authorizationButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            authorizationButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
         ])
     }
 
-    @objc private func signInButtonTapped() {
-        let homeVC = HomeViewController()
-        navigationController?.pushViewController(homeVC, animated: true)
+    @objc func handleAuthorizationAppleIDButtonPress() {
```

Comment:
```suggestion
    @objc private func handleAuthorizationAppleIDButtonPress() {
```

여기도 마찬가지로 private 붙일 수 있어요.